lib/src/components/NxTransferList/types.ts

```diff
@@ -39,3 +50,25 @@ export interface Props<T extends string | number = string> extends StatefulProps
   onAvailableItemsFilterChange: NxFilterInputProps['onChange'];
   onSelectedItemsFilterChange: NxFilterInputProps['onChange'];
 }
+
+export const propTypes: ValidationMap<Props<string | number>> = {
+  allItems: PropTypes.arrayOf(PropTypes.shape({
+    id: PropTypes.oneOfType([
+      PropTypes.string.isRequired,
+      PropTypes.number.isRequired
+    ]).isRequired,
+    displayName: PropTypes.string.isRequired
+  })).isRequired as Validator<DataItem<string | number>[]>,
```

Comment:
Unecessary cast?

Reply:
> Unecessary cast?

Thanks! I was missing an `.isRequired`.
This was fixed in this commit:
0f0c1c1799991ec9d4c9e64eca29b9c86c4061c0